pkg/collector/collector.go

```diff
@@ -54,6 +54,25 @@ type VersionInfo struct {
 
 var versionRegex = regexp.MustCompile(`ClamAV+\s([0-9.]*)\/?([0-9.]*)\/?(.*)`)
 
+func GetStat(matches *[][]string, index uint) float64 {
```

Comment:
I don't think this needs to accept a pointer. The slice is already a pointer under-the-hood